martin/src/config.rs

```diff
@@ -71,6 +73,17 @@ impl Config {
         let mut res = UnrecognizedValues::new();
         copy_unrecognized_config(&mut res, "", &self.unrecognized);
 
+        if self
```

Comment:
parsing here should be the same as is done for `x-rewrite-url` -- probably we need a dedicated function for that, used from both places.  The error message should probably say something like `Base path must be a valid URL path, and must begin with a '/' symbol`.